### DISCUSSION TITLE
Issue pulling pinned versions of PIXI

### DISCUSSION THREAD

**Discussion by RichardGale** [pixijs/pixijs#8256] [General]
### Expected Behavior
If I specify a version for pixi.js in my package.json, and it worked previously, I would expect it to work if I re-install.

### Current Behavior
I pinned by pixi.js to version 6.2.2 by specifying pixi.js like this pixi.js@6.2.2 in my package.json

I deleted my node_modules and re-installed with the following errors:
ERROR in ../node_modules/pixi.js/dist/esm/pixi.js 11:0-59
Module not found: Error: Can't resolve '@pixi/accessibility' in '/Users/richardgale/git/hidden/build/node_modules/pixi.js/dist/esm'
 @ ../../src/index.js 3:0-32 145:16-32

ERROR in ../node_modules/pixi.js/dist/esm/pixi.js 13:0-55
Module not found: Error: Can't resolve '@pixi/interaction' in '/Users/richardgale/git/hidden/build/node_modules/pixi.js/dist/esm'
 @ ../../src/index.js 3:0-32 145:16-32

ERROR in ../node_modules/pixi.js/dist/esm/pixi.js 19:0-40
Module not found: Error: Can't resolve '@pixi/extract' in '/Users/richardgale/git/hidden/build/node_modules/pixi.js/dist/esm'
 @ ../../src/index.js 3:0-32 145:16-32

ERROR in ../node_modules/pixi.js/dist/esm/pixi.js 23:0-90
Module not found: Error: Can't resolve '@pixi/compressed-textures' in '/Users/richardgale/git/hidden/build/node_modules/pixi.js/dist/esm'
 @ ../../src/index.js 3:0-32 145:16-32

ERROR in ../node_modules/pixi.js/dist/esm/pixi.js 25:0-60
Module not found: Error: Can't resolve '@pixi/particle-container' in '/Users/richardgale/git/hidden/build/node_modules/pixi.js/dist/esm'
 @ ../../src/index.js 3:0-32 145:16-32

ERROR in ../node_modules/pixi.js/dist/esm/pixi.js 27:0-40
Module not found: Error: Can't resolve '@pixi/prepare' in '/Users/richardgale/git/hidden/build/node_modules/pixi.js/dist/esm'
 @ ../../src/index.js 3:0-32 145:16-32

ERROR in ../node_modules/pixi.js/dist/esm/pixi.js 29:0-54
Module not found: Error: Can't resolve '@pixi/spritesheet' in '/Users/richardgale/git/hidden/build/node_modules/pixi.js/dist/esm'
 @ ../../src/index.js 3:0-32 145:16-32

ERROR in ../node_modules/pixi.js/dist/esm/pixi.js 31:0-59
Module not found: Error: Can't resolve '@pixi/sprite-tiling' in '/Users/richardgale/git/hidden/build/node_modules/pixi.js/dist/esm'
 @ ../../src/index.js 3:0-32 145:16-32

ERROR in ../node_modules/pixi.js/dist/esm/pixi.js 33:0-53
Module not found: Error: Can't resolve '@pixi/text-bitmap' in '/Users/richardgale/git/hidden/build/node_modules/pixi.js/dist/esm'
 @ ../../src/index.js 3:0-32 145:16-32

ERROR in ../node_modules/pixi.js/dist/esm/pixi.js 39:0-62
Module not found: Error: Can't resolve '@pixi/filter-color-matrix' in '/Users/richardgale/git/hidden/build/node_modules/pixi.js/dist/esm'
 @ ../../src/index.js 3:0-32 145:16-32

ERROR in ../node_modules/pixi.js/dist/esm/pixi.js 40:0-63
Module not found: Error: Can't resolve '@pixi/filter-displacement' in '/Users/richardgale/git/hidden/build/node_modules/pixi.js/dist/esm'
 @ ../../src/index.js 3:0-32 145:16-32

ERROR in ../node_modules/pixi.js/dist/esm/pixi.js 41:0-47
Module not found: Error: Can't resolve '@pixi/filter-fxaa' in '/Users/richardgale/git/hidden/build/node_modules/pixi.js/dist/esm'
 @ ../../src/index.js 3:0-32 145:16-32

ERROR in ../node_modules/pixi.js/dist/esm/pixi.js 42:0-49
Module not found: Error: Can't resolve '@pixi/filter-noise' in '/Users/richardgale/git/hidden/build/node_modules/pixi.js/dist/esm'
 @ ../../src/index.js 3:0-32 145:16-32

ERROR in ../node_modules/pixi.js/dist/esm/pixi.js 43:0-37
Module not found: Error: Can't resolve '@pixi/mixin-cache-as-bitmap' in '/Users/richardgale/git/hidden/build/node_modules/pixi.js/dist/esm'
 @ ../../src/index.js 3:0-32 145:16-32

ERROR in ../node_modules/pixi.js/dist/esm/pixi.js 44:0-39
Module not found: Error: Can't resolve '@pixi/mixin-get-child-by-name' in '/Users/richardgale/git/hidden/build/node_modules/pixi.js/dist/esm'
 @ ../../src/index.js 3:0-32 145:16-32

ERROR in ../node_modules/pixi.js/dist/esm/pixi.js 45:0-41
Module not found: Error: Can't resolve '@pixi/mixin-get-global-position' in '/Users/richardgale/git/hidden/build/node_modules/pixi.js/dist/esm'
 @ ../../src/index.js 3:0-32 145:16-32

ERROR in ../node_modules/pixi.js/dist/esm/pixi.js 54:0-38
Module not found: Error: Can't resolve '@pixi/sprite-animated' in '/Users/richardgale/git/hidden/build/node_modules/pixi.js/dist/esm'
 @ ../../src/index.js 3:0-32 145:16-32

ERROR in ../node_modules/pixi.js/dist/esm/pixi.js 55:0-27
Module not found: Error: Can't resolve '@pixi/text' in '/Users/richardgale/git/hidden/build/node_modules/pixi.js/dist/esm'
 @ ../../src/index.js 3:0-32 145:16-32

If I remove the version and re-install everything is ok. The version pulled was 6.3.0

### Environment

- **`pixi.js` version**: _e.g. 4.7.1_
- **Browser & Version**: _e.g. Chrome 67_
- **OS & Version**: _e.g. Ubuntu 18.04_
- **Running Example**: _url_

**Comment by ShukantPal** [comment]
@RichardGale This seems like an issue in your setup. @pixi/accessibility@6.2.2 [exists](https://www.npmjs.com/package/@pixi/accessibility/v/6.2.2).
